core/geometry/src/curve/LineString3d.ts

```diff
@@ -713,6 +713,32 @@ export class LineString3d extends CurvePrimitive implements BeJSONFunctions {
       return sum;
     }
   }
+/** Compute the range of points between fractional positions on the linestring. */
+public override rangeBetweenFractions(fraction0: number, fraction1: number): Range3d {
+  const range = Range3d.create ();
+  if (fraction1 === fraction0 || this.points.length < 1)
+    return range;
```

Comment:
If fractions are equal, we should return 1-point range like default implementation.